frontend/packages/dev-console/integration-tests/support/step-definitions/customization/customization-of-catalog-addPage-form-view.ts

```diff
@@ -0,0 +1,228 @@
+        } else {
+          cy.wrap($el).click();
+        }
+      });
+      cy.get('.co-search-group__pin-toggle').should('be.visible').click();
+    }
+  });
+});
+
+When('user clicks on cluster', () => {
+  cy.byTestID('cluster').should('be.visible').click();
+});
+
+When('user opens customization', () => {
+  cy.byLegacyTestID('actions-menu-button').should('be.visible').click();
+  cy.byTestActionID('Customize').should('be.visible').click();
+});
+
+Given('user is at customization', () => {
+  navigateTo(devNavigationMenu.Consoles);
+  cy.byLegacyTestID('actions-menu-button').should('be.visible').click();
+  cy.byTestActionID('Customize').should('be.visible').click();
+});
+
+When('user clicks on Developer tab', () => {
+  cy.get(customizationPO.role.presentation).contains('Developer').should('be.visible').click();
+});
+
+When('user disables all the items in Developer Catalog', () => {
+  cy.byTestID(customizationPO.developerCatalog.formSection)
+    .find(customizationPO.developerCatalog.addAllAction)
+    .scrollIntoView()
+    .should('be.visible')
+    .click();
+});
+
+When('user disables "HelmChart" the items in Developer Catalog', () => {
+  cy.byTestID(customizationPO.developerCatalog.formSection)
+    .find(customizationPO.developerCatalog.removeAllAction)
+    .scrollIntoView()
+    .click();
+
+  cy.byTestID(customizationPO.developerCatalog.formSection)
+    .find(customizationPO.developerCatalog.availableSearchInput)
+    .type('Helm Charts');
+
+  cy.byTestID(customizationPO.developerCatalog.formSection)
+    .find(customizationPO.role.option)
+    .contains('Helm Charts')
+    .scrollIntoView()
+    .should('be.visible')
+    .click();
+
+  cy.byTestID(customizationPO.developerCatalog.formSection)
+    .find(customizationPO.developerCatalog.addSelectedAction)
+    .scrollIntoView()
+    .should('be.visible')
+    .click();
+});
+
+Given('user "HelmChart" in enabled and disables everything', () => {
+  cy.byTestID(customizationPO.developerCatalog.formSection)
+    .find(customizationPO.developerCatalog.addAllAction)
+    .scrollIntoView()
+    .then(($el) => {
+      if ($el.is(':visible')) {
+        cy.wrap($el).click();
+      }
+    });
+
+  cy.byTestID(customizationPO.developerCatalog.formSection)
+    .find(customizationPO.developerCatalog.choosenSearchInput)
+    .type('Helm Chart');
+
+  cy.byTestID(customizationPO.developerCatalog.formSection)
+    .find(customizationPO.role.option)
+    .contains('Helm Charts')
+    .click();
+
+  cy.byTestID(customizationPO.developerCatalog.formSection)
+    .find(customizationPO.developerCatalog.removeSelectedAction)
+    .should('be.visible')
+    .click();
+});
+
+Given('user disables all the items in Add page', () => {
+  cy.byTestID(customizationPO.addPage.formSection)
+    .find(customizationPO.addPage.addAllAction)
+    .scrollIntoView()
+    .should('be.visible')
+    .click();
+});
+
+Given('user disables "Import from Git" the items in Developer Catalog', () => {
```

Comment:
Don't use items in "" directly here instead use a variable 
```suggestion
Given('user disables {string} the items in Developer Catalog', (item: string) => {
```

Reply:
Sure, made all the changes.